core/settings/geonode_queue_settings.py

```diff
@@ -0,0 +1,21 @@
+from kombu import Exchange, Queue
+
+geoserver_exchange = Exchange("geonode", type="topic", durable=False)
+
+queue_all_events = Queue("broadcast", geoserver_exchange, routing_key="#")
```

Comment:
You don't have to manually create all this queue. But I think it's ok as long as it doesn't broke.
It's hard to see which one is using which if there are too many queues.